tests/unittests/GraphOptzTest.cpp

```diff
@@ -203,11 +203,15 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConv) {
   EXPECT_EQ(F_->getNodes().size(), 3);
 
   ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
+  optimizedF_ = optimizeFunction(F_);
+  EXPECT_EQ(optimizedF_->getNodes().size(), 2);
 
-  ASSERT_EQ(A->getNumUsers(), 1);
-  Node *newCV = A->getUsers().begin()->getUser();
+  bindings_.allocate(mod_.getPlaceholders());
+  bindings_.get(A)->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  checkNumericalEquivalence();
+
+  ASSERT_EQ(A->getNumUsers(), 2);
+  Node *newCV = std::next(A->getUsers().begin(), 1)->getUser();
```

Comment:
Is the iteration order guaranteed?  Maybe it would be more futureproof to loop through the nodes in `optimizedF_` and find the `ConvolutionNode` that is `!= CV`?

Reply:
Done, but I'm not fully familiar with these objects, so there might be a cleaner way to do this with the relationship functions.